free-2.md

```diff
@@ -58,12 +105,20 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think they'd be spending a lot of time making language design decisions, but also a fair amount of time implementing the language. One of the interesting decisions they'll have to make is figuring out what the deliverable _for this class_ will be - we obviously won't have access to an automated coffee machine or alcohol mixer (do those even exist?) to try it out on, so there would have to be some other deliverable to present when someone runs a program in this language.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I'd call this a medium sized idea. Implementing it is going to be difficult (because there will be a lot of language design choices to make), but I think it'll be manageable over the course of the semester.
+
+If I choose to attempt to monetize it, **that** would be ambitious. 
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+I think it would be a great idea for a project for me, because I'm very excited about coffee and I think it could ultimately be a really useful language. The downsides of this project are that it's a bit on the large side and it would be hard to work out what the deliverable of the project would be. 
+
```

Comment:
I think it's an interesting idea. As you mentioned, it would take some work to figure out what happens when a program runs: would it perhaps compile to a finite-state machine that could presumably then be executed by a physical machine? It would also be interesting to think about operations that may or may not be commutative (if you add the milk before the coffee, is that the same as adding the coffee before the milk?). How would one specify that the order of operations is important or not important, etc.?